docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+calibration_list = processor.list_calibration(START_SECONDS, END_SECONDS)
+
+# If you know the job-id, you can retrieve the calibration that the job used.
+job = engine.get_job("projects/" + PROJECT_ID
+                   + "/programs/"+PROGRAM_ID
+                   + "/jobs/" + JOB_ID)
+job_calibration = cg.EngineJob(cg.JobConfig(), job, engine).get_calibration()
+
+# The calibration can be iterated through using something like the following.
+for metric_name in latest_calibration:
+  print(metric_name)
+  print('------')
+  for qubit_or_pair in latest_calibration[metric_name]:
+     # Note that although the value is often singular,
+     # the metric_value is of the type list and can have multiple values     
+     metric_value = latest_calibration[metric_name][qubit_or_pair]
+     print(f'{qubit_or_pair} = {metric_value}')
+```
+
+Calibration metrics will also soon be available from the 
+[Google Cloud Platform Console](https://console.cloud.google.com).
+
+## Average, Pauli and Incoherent Error
+
+Several metrics below define average error, pauli error and incoherent error.
+This section explains the difference between each of those metrics.
+
+The average error is equal to one minus fidelity averaged over all possible
+input states.
+
+Pauli error defines decoherence of a single qubit in one of the Pauli channels
+X, Y, or Z.  If the errors are distributed in a uniform distribution across all
+three axes, the probability of applying an erroneous Pauli gate X, Y, or Z will
+be the Pauli error divided by three.  The Pauli error and average error are
+related by a multiplicative factor based on the number of qubits.
+
+See Table 1 on page 11 of the
+[https://arxiv.org/abs/1910.11333](Supplementry Information)
+document for a description and comparison between average error, Pauli error,
+and depolarization error.
+
+The incoherent error is the "unitarity" of the gate or cycle.
+This is defined as the decay rate per gate (or cycle) of the
+[Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics))
+when fit to an exponential curve.  This rate has been scaled to match the
+average error per Clifford gate (or per 2-qubit cycle).
+For more about purity benchmarking, see Section 6.3 of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf).
+
+The purity error can be interpreted as a measure of the incoherent error,
+such as those caused by stochastic processes such as relaxation.  The average
+error can be interpreted as containing both this incoherent error as well as
+the coherent error resulting from improper control or calibration of the device.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the average error by small amounts.
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+**Note that the metric names below are subject to change without notice.**
```

Comment:
Do we plan to freeze them at some point, e.g. cirq 1.0? Would it make sense to promise this here?

Reply:
I don't think there is a plan to freeze this, and they will likely change drastically as devices change and evolve.  For instance, if the number of qubits increases, we may no longer be able to provide isolated metrics.  I don't think we want to make any promises beyond the promise that they will almost definitely change.

Reply:
As usual, this is unlikely to stop users from writing code that depends on this (e.g. predictive models for circuit fidelity). That said, I'm glad we're reserving the right to change this.